caput/misc/__init__.py

```diff
@@ -46,15 +45,15 @@ def __call__(self, *args, **kwargs):
 
             return arr
 
-        def __get__(self, obj, type=None):
+        def __get__(self, obj, type_=None):
```

Comment:
TIL that the norm is to append an `_` (not prepend, which is what I thought it was.

Reply:
prepend on members means "please treat this as private", so it would confuse to use that

Reply:
Hmmm... I don't typically mind examples like this that mask builtin names. I mean, we're all adults and we can see that it's fine in here. However, as a compromise how about we just rename it to something else without an underscore. The python docs on descriptors refer to this parameter as `objtype` which seems good enough to me, and doesn't have the awkward trailing underscore.